docs/content/config/advanced/ipv6.md

```diff
@@ -2,47 +2,14 @@
 title: 'Advanced | IPv6'
 ---
 
-## Background
+## Not Available
+docker-mailserver doesn't currently support iPv6 networking! Trying to get mails using iPv6 will result with postfix not being able to find your hostname.
 
-If your container host supports IPv6, then `docker-mailserver` will automatically accept IPv6 connections by way of the docker host's IPv6. However, incoming mail will fail SPF checks because they will appear to come from the IPv4 gateway that docker is using to proxy the IPv6 connection (`172.20.0.1` is the gateway).
-
-This can be solved by supporting IPv6 connections all the way to the `docker-mailserver` container.
-
-## Setup steps
-
-```diff
-+++ b/serv/docker-compose.yml
-@@ -1,4 +1,4 @@
--version: '2'
-+version: '2.1'
-
-@@ -32,6 +32,16 @@ services:
-
-+  ipv6nat:
-+    image: robbertkl/ipv6nat
-+    restart: always
-+    network_mode: "host"
-+    cap_add:
-+      - NET_ADMIN
-+      - SYS_MODULE
-+    volumes:
-+      - /var/run/docker.sock:/var/run/docker.sock:ro
-+      - /lib/modules:/lib/modules:ro
-
-@@ -306,4 +316,13 @@ networks:
-
-+  default:
-+    driver: bridge
-+    enable_ipv6: true
-+    ipam:
-+      driver: default
-+      config:
-+        - subnet: fd00:0123:4567::/48
-+          gateway: fd00:0123:4567::1
-```
+## Fix
+To fix the issue, basicly remove the AAAA record for your mailserver. This will prevent email senders from using iPv6.
```

Comment:
> To fix the issue, basicly remove the AAAA record for your mailserver.

Haha, I'm all for that ❤️ This is probably not something someone who needs to use IPv6 wants to hear..

Reply:
I didn't want to hear this too but that was the only fix I was able to find. :)